chain-addr/src/lib.rs

```diff
@@ -418,6 +416,21 @@ impl std::str::FromStr for AddressReadable {
     }
 }
 
+impl SerializedSize for Address {
+    fn serialized_size(&self) -> usize {
+        0_u8.serialized_size()
```

Comment:
this looks a bit weird, tbh... I know it makes no difference, but maybe `self.to_kind_value().serialized_size()` reads a bit better.

Reply:
fixed